lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
```

Comment:
Now `flow` and `composition` are just aliases, in future I want to upgrade `flow` to functor with additional methods (more functionality and overhead) and leave `composition` as a simple async composition.